swan-lake/learn/security/authentication-and-authorization.md

```diff
@@ -1,530 +1,309 @@
 > Accept: */*
 > 
 < HTTP/1.1 401 Unauthorized
+< content-length: 0
+< server: ballerina
 < content-type: text/plain
 < 
-Authentication failure
 ```
 
-If a request is made with a valid, signed JWT, but without the expected `scope`, an authorization failure will occur. An example of a JWT without the `scope` attribute is as follows.
-
-```
-{
-  "sub": "ballerina",
-  "iss": "ballerina",
-  "exp": 2818415019,
-  "iat": 1524575019,
-  "jti": "f5aded50585c46f2b8ca233d0c2a3c9d",
-  "aud": [
-    "ballerina",
-    "Ballerina.org",
-    "ballerina.io"
-  ]
-}
-```
+If a request is made with valid authentication information, but the authenticated user does not have the required permission, an authorization failure will occur.
 
 ```
-curl -k -v https://localhost:9091/hello -H "Authorization:Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJiYWxsZXJpbmEiLCJpc3MiOiJiYWxsZXJpbmEiLCJleHAiOjI4MTg0MTUwMTksImlhdCI6MTUyNDU3NTAxOSwianRpIjoiZjVhZGVkNTA1ODVjNDZmMmI4Y2EyMzNkMGMyYTNjOWQiLCJhdWQiOlsiYmFsbGVyaW5hIiwiYmFsbGVyaW5hLm9yZyIsImJhbGxlcmluYS5pbyJdfQ.X2mHWCr8A5UaJFvjSPUammACnTzFsTdre-P5yWQgrwLBmfcpr9JaUuq4sEwp6to3xSKN7u9QKqRLuWH1SlcphDQn6kdF1ZrCgXRQ0HQTilZQU1hllZ4c7yMNtMgMIaPgEBrStLX1Ufr6LpDkTA4VeaPCSqstHt9WbRzIoPQ1fCxjvHBP17ShiGPRza9p_Z4t897s40aQMKbKLqLQ8rEaYAcsoRBXYyUhb_PRS-YZtIdo7iVmkMVFjYjHvmYbpYhNo57Z1Y5dNa8h8-4ON4CXzcJ1RzuyuFVz1a3YL3gWTsiliVmno7vKyRo8utirDRIPi0dPJPuWi2uMtJkqdkpzJQ"
+curl -k -v https://localhost:9090/foo/bar -H "Authorization: Basic <token>"
 
-> GET /hello HTTP/1.1
-> Host: localhost:9091
+> GET /foo/bar HTTP/1.1
+> Host: localhost:9090
 > User-Agent: curl/7.47.0
 > Accept: */*
-> Authorization:Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJiYWxsZXJ
-pbmEiLCJpc3MiOiJiYWxsZXJpbmEiLCJleHAiOjI4MTg0MTUwMTksImlhdCI6MTUyNDU3NTAxOSwian
-RpIjoiZjVhZGVkNTA1ODVjNDZmMmI4Y2EyMzNkMGMyYTNjOWQiLCJhdWQiOlsiYmFsbGVyaW5hIiwiY
-mFsbGVyaW5hLm9yZyIsImJhbGxlcmluYS5pbyJdfQ.X2mHWCr8A5UaJFvjSPUammACnTzFsTdre-P5y
-WQgrwLBmfcpr9JaUuq4sEwp6to3xSKN7u9QKqRLuWH1SlcphDQn6kdF1ZrCgXRQ0HQTilZQU1hllZ4c
-7yMNtMgMIaPgEBrStLX1Ufr6LpDkTA4VeaPCSqstHt9WbRzIoPQ1fCxjvHBP17ShiGPRza9p_Z4t897
-s40aQMKbKLqLQ8rEaYAcsoRBXYyUhb_PRS-YZtIdo7iVmkMVFjYjHvmYbpYhNo57Z1Y5dNa8h8-4ON4
-CXzcJ1RzuyuFVz1a3YL3gWTsiliVmno7vKyRo8utirDRIPi0dPJPuWi2uMtJkqdkpzJQ
+> Authorization: Basic <token>
 >
 
 < HTTP/1.1 403 Forbidden
+< content-length: 0
+< server: ballerina
 < content-type: text/plain
 <
-Authorization failure
 ```
 
-A request with a correct "scope" attribute will result in a successful invocation. An example of a JWT that has the correct "scope" attribute is as follows.
-
-```
-{
-  "sub": "ballerina",
-  "iss": "ballerina",
-  "exp": 2818415019,
-  "iat": 1524575019,
-  "jti": "f5aded50585c46f2b8ca233d0c2a3c9d",
-  "aud": [
-    "ballerina",
-    "ballerina.org",
-    "ballerina.io"
-  ],
-  "scope": "hello"
-}
-```
+A request is made with valid authentication information, and the authenticated user has the required permission, will result in a successful invocation.
```

Comment:
```suggestion
If a request is made with valid authentication information and the authenticated user has the required permission, this will result in a successful invocation.
```